src/components/utils/popover.ts

```diff
@@ -720,4 +720,20 @@ export default class Popover extends EventsDispatcher<PopoverEvent> {
 
     return popoverPotentialTopEdge < scopeElementRect.top || popoverPotentialBottomEdge <= bottomEdgeForComparison;
   }
+
+  /**
+   * Returns popover item with filled in title property if missing
+   *
+   * @param item - item to fill in title of
+   */
+  private fillItemTitle(item: PopoverItem): PopoverItem {
+    if (item.title !== undefined) {
+      return item;
+    }
+
+    return {
+      ...item,
+      title: capitalize(item.name),
```

Comment:
do not use cypress-name for business logic. We should consider renaming this property to `cyName`